rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -841,6 +854,31 @@ def test_exec(config, ssh_con):
                                     f"bucket check fix did not removed orphan objects on a bucket {bucket.name}"
                                 )
 
+                if config.test_ops.get("test_multipart_race_complete_abort"):
+                    utils.exec_shell_cmd(f"radosgw-admin gc process --include-all")
+                    log.info(
+                        "sleeping for 10 seconds before verifying objects download"
+                    )
+                    time.sleep(10)
+                    log.info("downloading all objects in bucket: %s" % bucket.name)
+                    objects = s3lib.resource_op(
+                        {"obj": bucket, "resource": "objects", "args": None}
+                    )
+                    log.info("objects :%s" % objects)
```

Comment:
@hmaheswa  can we use format in log messages

Reply:
done @anrao19 
pls re-review